src/packages/migrate/src/__tests__/DbPush.test.ts

```diff
@@ -0,0 +1,116 @@
+process.env.GITHUB_ACTIONS = '1'
```

Comment:
A little comment why this is needed would be nice

Reply:
It's to get into the CI "mode" for snapshots, until we have some tests that can test both